Dockerfiles/ccp-openshift-scan/Dockerfile

```diff
@@ -4,4 +4,4 @@ ADD scanning /opt/scanning
 
 VOLUME ["/opt/scanning"]
 
-CMD ["/bin/tail", "-f", "/dev/null"]
+CMD ["/bin/sleep", "infinity"]
```

Comment:
What's the problem with `tail -f /dev/null`? Is `sleep infinity` a better approach? Is there a reference for it?

Reply:
I tried running both the commands and used `htop` to observe them. `tail` is not showing the behavior of continuous processing. I'm saying so because CPU column doesn't show any spike. I even did

```bash
$ for i in `seq 100`; do echo $i > /dev/null; sleep 1; done
``` 
to see if there's a spike. But there wasn't any.

It's how we think tail works that's probably making us believe so.